ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/permit-simulation.tsx

```diff
@@ -21,7 +23,10 @@ import {
 import { SignatureRequestType } from '../../../../../types/confirm';
 import useTokenExchangeRate from '../../../../../../../components/app/currency-input/hooks/useTokenExchangeRate';
 import { IndividualFiatDisplay } from '../../../../simulation-details/fiat-display';
-import { formatNumber } from '../../../utils';
+import {
+  formatAmount,
+  formatAmountMaxPrecision,
+} from '../../../../simulation-details/formatAmount';
```

Comment:
We could move this to a higher level util but I think this makes sense in this context since we're also a simulation component 👍